samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -57,7 +59,7 @@ public ExecutionPlanner(Config config, StreamManager streamManager) {
     this.streamManager = streamManager;
   }
 
-  public JobGraph plan(StreamGraph streamGraph) throws Exception {
+  public StreamPlan plan(StreamGraph streamGraph) throws Exception {
```

Comment:
StreamingPlan seems fine, but StreamPlan doesn't make sense by itself. @jmakes, any thoughts? We can also rename later, not a blocker.